src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -205,6 +205,15 @@ public static GitHub connectAnonymously() throws IOException {
     }
 
     /**
+     * Connects to GitHub Enterprise anonymously.
```

Comment:
Analogous to `GitHub.connectAnonymously()`. [Reference](https://github.com/kohsuke/github-api/blob/master/src/main/java/org/kohsuke/github/GitHub.java#L203)